firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -128,7 +117,7 @@ public static FirebaseInstallations getInstance(@NonNull FirebaseApp app) {
   public Task<String> getId() {
     return Tasks.call(executor, this::getPersistedFid)
         .continueWith(orElse(this::createAndPersistNewFid))
-        .onSuccessTask(this::registerFidIfNecessary);
+        .onSuccessTask(persistedFidEntry -> registerFidIfNecessary(persistedFidEntry));
```

Comment:
this could be a method ref

Reply:
I changed this when I was trying listener logic.